docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -142,6 +142,7 @@ Along with the HTTP error code, our API can also return more detailed error code
 | 30013  | Maximum number of guild channels reached (500)                                                                                |
 | 30015  | Maximum number of attachments in a message reached (10)                                                                       |
 | 30016  | Maximum number of invites reached (1000)                                                                                      |
+| 30031  | Guild already has a template                                                                                                  |
```

Comment:
fyi, there's also: `UNKNOWN_GUILD_TEMPLATE = ('Unknown guild template', 10057)`

Reply:
already added that in #1712